### PR TITLE
#179: Allow adding a parcel with packing date set to today

### PR DESCRIPTION
**Pull request by AmeliaIsCoding**
## What's changed
It is now possible to add a new parcel with the packing date set to today.

## Screenshots / Videos
| Before     | After      |
|------------|------------|
|image|image|

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

No DB changes

### CODE REVIEW

**Review by AmeliaIsCoding — COMMENTED** [review]
self-review

File: src/components/Form/formFunctions.ts [hunk 1]
```diff
@@ -173,7 +173,7 @@ export const onChangeDate = (
         return;
     }
 
-    const today = dayjs();
+    const today = dayjs().startOf("day");
```
Comment (rickaw) [inline src/components/Form/formFunctions.ts:176]:
```suggestion
    const ealiestPossibleDateTime = dayjs().startOf("day");
```

**Review by rickaw — APPROVED** [review]
Nice!

File: src/components/Form/formFunctions.ts (hunk 1 above)
Comment (AmeliaIsCoding) [reply src/components/Form/formFunctions.ts]: sure

**Pull request merged**